docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -287,10 +287,14 @@ They are in default namespace, named as kubernetes-{network}, for the kubernetes
 
 Query for default-ns scoped kubernetes-{network} shall get back the proper content based on the cluster kubernetes endpoints object. System does not duplicate such endpoints; instead it derives content based on the cluster kubernetes endpoints. This would incurs quite some code change to kube-apiserver.
 
+For now, get verb with this kind of EP is supported; however, list/watch verb won't include per-network kubernetes EP. For controller that needs to use these EPs, one workaround is deriving from the cluster scoped EP, besides using get verb in this special case.
```

Comment:
minor: For controller that needs -> For a controller that needs

Reply:
will update as suggested.